<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicGame</class>
 <widget class="QMainWindow" name="LogicGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeuroGame - Aritmética</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 245, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="timer_text">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>37</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sound">
    <property name="geometry">
     <rect>
      <x>1227</x>
      <y>38</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/volume_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="music">
    <property name="geometry">
     <rect>
      <x>1176</x>
      <y>38</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/music.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>67</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/star.png);
color: rgb(221, 122, 179);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="intructions">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>¡Resuelve las operaciones!</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>1031</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="previous_addend_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="previous_addend_2">
     <property name="geometry">
      <rect>
       <x>445</x>
       <y>119</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="previous_equal">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>120</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="previous_result">
     <property name="geometry">
      <rect>
       <x>625</x>
       <y>118</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_equal">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>210</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_addend_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>210</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_addend_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="incoming_equal">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>310</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="incoming_addend_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>311</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="incoming_addend_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="previous_opeation">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_operation">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="incoming_opeation">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>310</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 57, 132);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>210</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 252, 252);
color: rgb(69, 69, 229);</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>189</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(69, 69, 229);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="feedback">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>200</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>previous_addend_1</zorder>
    <zorder>previous_addend_2</zorder>
    <zorder>previous_equal</zorder>
    <zorder>previous_result</zorder>
    <zorder>current_equal</zorder>
    <zorder>current_addend_2</zorder>
    <zorder>current_addend_1</zorder>
    <zorder>incoming_equal</zorder>
    <zorder>incoming_addend_2</zorder>
    <zorder>incoming_addend_1</zorder>
    <zorder>previous_opeation</zorder>
    <zorder>current_operation</zorder>
    <zorder>incoming_opeation</zorder>
    <zorder>user_input</zorder>
    <zorder>feedback</zorder>
   </widget>
   <widget class="QLabel" name="operation_number">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>660</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 122, 179);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>timer_text</zorder>
   <zorder>sound</zorder>
   <zorder>music</zorder>
   <zorder>score</zorder>
   <zorder>intructions</zorder>
   <zorder>operation_number</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
